myip.go

```diff
@@ -49,3 +50,24 @@ func GetIP() (net.IP, error) {
 
 	return nil, fmt.Errorf("Unable to find IP")
 }
+
+// GetIPUsingIfConfig_Me uses the ifconfig.me web service to get your public IP. It returns your public IP or an error.
+func GetIPUsingIfConfig_Me() (net.IP, error) {
+	resp, err := http.Get(ifconfig.me)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to read Body")
+	}
+
+	ipS := string(body)
+	ip := net.ParseIP(ipS)
```

Comment:
Consider combining the line above like this:

ip := net.ParseIP(string(body))

I think it is still pretty clear and lets you avoid the awkward variable name.

Reply:
Done.